test/operator/installation.go

```diff
@@ -25,6 +31,60 @@ func InstallFromSource(withCsi bool) features.Func {
 	return manifests.InstallFromFiles(paths)
 }
 
+func InstallViaMake() features.Func {
+	return func(ctx context.Context, t *testing.T, environmentConfig *envconf.Config) context.Context {
+		rootDir := getSourceRootDir()
+
+		runBuildAndManifests(rootDir, t)
+
+		platform := kubeobjects.ResolvePlatformFromEnv()
+		makeTarget := "deploy"
+		switch platform {
+		case kubeobjects.Openshift:
+			makeTarget = strings.Join([]string{makeTarget, "openshift"}, "/")
+		case kubeobjects.Kubernetes:
+			makeTarget = strings.Join([]string{makeTarget, "kubernetes"}, "/")
+		default:
+			t.Fatal("failed to install the operator via the make command as no correct platform was set")
+			return nil
+		}
+
+		err := exec.Command("make", "-C", rootDir, makeTarget).Run()
+		if err != nil {
+			t.Fatal("failed to install the operator via the make command", err)
+			return nil
+		}
+
+		return ctx
+	}
+}
+
+func runBuildAndManifests(rootDir string, t *testing.T) {
+	err := exec.Command("make", "-C", rootDir, "build").Run()
+	if err != nil {
+		t.Fatal("failed to install the operator via the make command", err)
+		return
+	}
+
+	err = exec.Command("make", "-C", rootDir, "manifests/branch").Run()
+	if err != nil {
+		t.Fatal("failed to install the operator via the make command", err)
+		return
+	}
+}
+
+func getSourceRootDir() string {
```

Comment:
[project.RootDir](https://github.com/Dynatrace/dynatrace-operator/blob/c49dfc1f643b15bda9e71b0517a911a5b87911f1/test/project/project.go#L10) function could be used instead of new function doing the same job

Reply:
thank you, did completely oversee this function.